datadog/dogstatsd/buffered_metrics.py

```diff
@@ -0,0 +1,70 @@
+import random
+from datadog.dogstatsd.metric_types import MetricType
+
+
+class BufferedMetric(object):
+    def __init__(self, name, tags, metric_type, max_metrics=0, specified_rate=1.0):
+        self.name = name
+        self.tags = tags
+        self.metric_type = metric_type
+        self.max_metrics = max_metrics
+        self.specified_rate = specified_rate
+        self.data = []
+        self.stored_metrics = 0
```

Comment:
Should these be set to `1`?

Reply:
The object will be instantiated immediately in the aggregator w/o a metric being added yet so i think 0 is fine